Updated docs added explanation. (#17751)
hashicorp/consul: website/content/commands/operator/raft.mdx

`Voter` is "true" or "false", indicating if the server has a vote in the Raft
configuration.

`Commit Index` is the last log index the server has a record of in its Raft log.

> `Trails Leader By` is the difference in commit index the server has from the leader.

Reviewer: ```suggestion
`Trails Leader By` is the number of commits a follower trails the leader by.
```

Author: I had enabled auto merge - please approve - https://github.com/hashicorp/consul/pull/17763
